sample-apps/spark/Dockerfile

```diff
@@ -7,7 +7,7 @@ COPY ./settings.gradle ./settings.gradle
 
 RUN gradle build
 RUN tar -xvf build/distributions/spark-sample-app-1.0.tar
-RUN wget https://github.com/aws-observability/aws-otel-java-instrumentation/releases/download/v0.12.1-aws.1/aws-opentelemetry-agent.jar
+RUN wget https://github.com/aws-observability/aws-otel-java-instrumentation/releases/download/v0.10.1-aws.1/aws-opentelemetry-agent.jar
```

Comment:
I thought only v0.13.0 doesn't work. Does v0.12.1 also has issue for emitting the compatible metrics?

Reply:
Same reason as in the other comment. I was testing it locally without java-agent and docker and `v0.12.1` is not compatible with the `IntervalMetricReader` builder. Will change this back.